src/main/resources/application-prod.yml

```diff
@@ -2,10 +2,10 @@ server:
   address: edview.unilu.ch
   port: 8443
   ssl:
-    key-store-type: PKCS12
-    key-store: classpath:certs/prod/httpd.keystore.p12
-    key-store-password: uni-lu-eft-2024
-    key-alias: unilu-eft
+    key-store-type: ${KS_TYPE}
+    key-store: /${SEC_LOCATION}/httpd.keystore.p12
```

Comment:
I'd make this entirely configurable if possible. If they ever switch keystore type, they would still have to name it `httpd.keystore.p12`., which does not make much sense.